etcdctl/command/util.go

```diff
@@ -221,6 +221,10 @@ func mustNewClient(c *cli.Context) client.Client {
 		err := hc.Sync(ctx)
 		cancel()
 		if err != nil {
+			if err == client.ErrNoEndpoints {
+				fmt.Fprintf(os.Stderr, "Failed to get any client endpoints when synchronizing through %s.\n", strings.Join(eps, ","))
+				fmt.Fprintf(os.Stderr, "Please ensure that etcd members can talk to each other through advertise peer urls, and each member sets its advertise client urls non-empty.\n")
```

Comment:
can we remove this log line? I do not think line provides any useful information.